tests/src/test/scala/com/nvidia/spark/rapids/StringFunctionSuite.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids
+
+class StringFunctionSuite extends SparkQueryCompareTestSuite {
```

Comment:
I tried to add the test cases for scalar-scalar input, but it seems that those cases are compiled-time evaluated because my code didn't run.

Reply:
To test scalar-scalar input, you would need to write a unit test, not something that goes through Spark's optimizer, i.e.: constructing a `GpuStringRepeat` instance directly and invoking `columnarEval` on it with two child expressions that will `columnarEval` to a `GpuScalar`.